src/style.css

```diff
@@ -0,0 +1,123 @@
+@font-face {
+  font-family: "Avenir";
+  src: url(fonts/Avenir-Book.ttf);
+  font-style: normal;
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-style: normal;
+  font-weight: 800;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+html {
+  font-family: Avenir, sans-serif;
+}
+
+.form {
+  position: relative;
+  margin-top: 20px;
+}
+
+.input {
+  width: 100%;
+}
+
+[class$="img"] {
+  position: absolute;
+}
```

Comment:
No need to use attribute selector, just set styles to one image and overlap them using other class